Jenkinsfile

```diff
@@ -0,0 +1,35 @@
+pipeline {
+
+    agent {
+        label '!platform:true && image_ver:18.4.0 && pkgsrc_arch:x86_64 && pi:20151126T062538Z && jenkins_agent:2'
+    }
+
+    options {
+        buildDiscarder(logRotator(numToKeepStr: '90'))
+        timestamps()
+    }
+
+    stages {
+        stage('check') {
+            steps{
+                sh('make check')
+            }
+        }
+        // avoid bundling devDependencies
+        stage('re-clean') {
+            steps {
+                sh('git clean -fdx')
+            }
+        }
```

Comment:
Aside: There is some npm pruning command that can be run to trim out devDependencies right? Eventually we should have that be part of the standard build steps in all the repos.

Reply:
I think the fact that we get different results if `make check` or `make test` is run before the image build is bug, but I'm not sure there is a universal fix for all of our quirky components.  Maybe with the node v6 era npm we can `prune`?